ui/pages/confirmations/send/send-content/add-recipient/domain-input.component.js

```diff
@@ -106,14 +114,19 @@ export default class DomainInput extends Component {
           {hasSelectedAddress ? (
             <>
               <div className="ens-input__wrapper__input ens-input__wrapper__input--selected">
+                <AvatarAccount
+                  variant={
+                    isBlockie
+                      ? AvatarAccountVariant.Blockies
+                      : AvatarAccountVariant.Jazzicon
+                  }
+                  address={selectedAddress}
+                  size={Size.SM}
+                  borderColor={BackgroundColor.backgroundDefault} // we currently don't have white color for border hence using backgroundDefault as the border
+                />
                 <div className="ens-input__selected-input__title">
```

Comment:
Correct me if I'm wrong, but I think we need to show `ENS ?? address` for the recipient, not the account name

Reply:
![Screenshot 2024-04-08 at 3 35 19 PM](https://github.com/MetaMask/metamask-extension/assets/100321200/46e776d7-2d74-441e-a519-12be48603213)

Reply:
Have this on the task list for the [revisions PR](https://github.com/MetaMask/metamask-extension/pull/23921)